### ISSUE TITLE
Missing crossorigin attribute on SVG image element

### ISSUE THREAD

**Issue by Ryuno-Ki** [mdn/browser-compat-data#9238]
TL;DR: What is the process to add data to a new attribute of a (SVG) element?

This is my first issue with BCD.

In order to complete a PR on mdn/content, I'd need to add a new entry to https://github.com/mdn/browser-compat-data/blob/f58673c23654744b36c3cae47a115aa474965566/svg/elements/image.json since the SVG spec got updated and thereby introducing a new attribute to SVG's `<image>`.

I'm only posessing a Sabayon Linux with Firefox and Chromium (Opera counts as Chromium?). How do I fill other browser/OS combinations? Can I mark them as unclear somehow?

# Background

@edemaine originally reported this in https://github.com/mdn/content/issues/1916
I have a PR ready to be merged in https://github.com/mdn/content/pull/2547

# Prior search

This might intersect with https://github.com/mdn/browser-compat-data/issues/7585 - but there, it is mentioning `FEImage` on SVG (I'm talking about `<image>` here).

**Comment by ddbeck** [comment]
Hi @Ryuno-Ki, thanks for your patience while I got to your questions. You're on the right track here to edit the `image.json` file and create a new `"crossorigin": { "__compat": … }` feature as a sibling to the other attributes in that file, then open a PR with the new feature (please refer back to this issue at that time).

> I'm only posessing a Sabayon Linux with Firefox and Chromium (Opera counts as Chromium?). How do I fill other browser/OS combinations? Can I mark them as unclear somehow?

You can indicate `"version_added"` support with a string (for the specific version that support was added), `true` (supported by exact version unknown), `null` (completely unknown), or `false` (known to be unsupported). For example, if it was supported in Chrome 88 and later, but not in 87, then you'd enter `"version_added": "88"`, but if you don't know the support status of Safari,  you'd enter `"version_added": null`.

If you have a minimal test case that you can share, we can also help fill in any gaps in the data that you haven't figured out yet.

> (Opera counts as Chromium?)

We track Chrome, Opera, Edge, etc. separately (they have varying version discrepancies). You can use [our browser data](https://github.com/mdn/browser-compat-data/blob/master/browsers/) (for example, [`opera.json`](https://github.com/mdn/browser-compat-data/blob/master/browsers/opera.json)) to work backwards from an Opera version to Blink then back again to the other browsers (I think this is a safe bet for this kind of feature). If you're unsure about how to do this, fill in the data for the browsers you can test and we can help you fill in the rest when you submit a PR.

Let me know if you have more questions.

**Comment by Ryuno-Ki** [comment]
Hey @ddbeck,

quick notice that I've only learned about this issue today.
I'm setting aside some time after Easter to come back to it (if someone else wants to work on it, please do).

**Comment by queengooborg** [comment]
This attribute has been added in #19999!
